front/src/modules/command-menu/components/WrapperCommandMenu.tsx

```diff
@@ -0,0 +1,248 @@
+import { useState } from 'react';
```

Comment:
Very nice, let's move this component within __stories__ folder as it's used for testing